src/main/java/dev/joopie/jambot/model/album/Album.java

```diff
@@ -22,7 +25,9 @@ public class Album extends BaseModel {
     public static final String EXTERNALID_FIELD = "external_id";
 
     private String name;
+    @Enumerated(EnumType.STRING)
     private AlbumGroup albumGroup;
+    @Enumerated(EnumType.STRING)
```

Comment:
Somehow, if set to EnumType.ORDINAL which is the standard, this sometimes results in ArrayOutOfBoundsException which I can not explain. By saving it this way, this gets fixed.